Advanced-Concepts/Two-Way-Client-Observers-old.md

```diff
@@ -1,57 +0,0 @@
----
-layout: page
-title: Two Way Client Observers
----
-{% include JB/setup %}
-
-**THE BELOW IS WRONG!!!!!!**
```

Comment:
was this file removed on purpose or a merge conflict? Because I see the .md file is gone, but there is html documentation generated for this.